docs/wxm-devices/D1-bundle/D1-troubleshooting.mdx

```diff
@@ -52,19 +54,7 @@ When the RF connection with the Weather Station is weak, the WG1200 Gateway will
 Low signal quality may be due to obstacles between the weather station and the WG1200 Gateway, such as walls, trees, or other buildings.
 :::
 
-### Temperature looks higher than expected.
-
-If the temperature is slightly higher (e.g., +2 - 5°C/35-41°F) than expected,
-then assure that the weather station is within a significant distance from heat sources
-like ventilation systems and active chimneys.
-Also make sure that the weather station is mounted on a 2m/6.5ft (or more) metallic pole and there are no obstacles close by reading [Best Practices for Proper Station Installation](/wxm-devices/D1-bundle/install-weather-station#best-practices-for-proper-station-installation).
-
-### Other variables, such as wind direction, wind speed, rain height and humidity do not align with an official weather reporting station
-
-Even weather stations, which are close to each other, may obtain slightly different measurements,
-especially for the variables of wind speed and wind direction. However, you should not expect significant deviations
-e.g., from a weather station, which is located within a distance of 1km/0.6mi.
-If there are deviations, please check [Best Practices for Proper Station Installation](/wxm-devices/D1-bundle/install-weather-station#best-practices-for-proper-station-installation) for guidelines about the installation of the weather station.
+<hr />
 
 ### Device Alert "Station Reported Abnormal Values".
```

Comment:
Add a : please and remove the dot at the end ----> `Device Alert: "Station Reported Abnormal Values"`